core/type/src/test/java/uk/gov/gchq/gaffer/types/function/ToTypeSubTypeValueTest.java

```diff
@@ -102,11 +99,11 @@ protected ToTypeSubTypeValue getInstance() {
 
     @Override
     protected Class[] getExpectedSignatureInputClasses() {
-        return new Class[]{Object.class};
+        return new Class[0];
```

Comment:
This seems wrong. The previous signature was correct. @m29827 Is this highlighting a bug elsewhere?

Reply:
By the looks of it the GafferFunctionTest doesn't run the same Signature tests as the FunctionTest one in Koryphe. Might be worth incorporating it to make for an easier transition once we upgrade the version of Junit in Koryphe.

Reply:
Hi @d47853 - agreed it's not running the Koryphe signature tests at the moment, when they are implemented the previous signature is the correct expected value.

Reply:
Hi thanks for spotting this - I've reverted to back to the same as develop branch - think the reason for this was from re-inheriting it whilst resolving a conflict